.github/workflows/deploy-production.yml

```
@@ -0,0 +1,91 @@
+        with:
+          eslint: true
+          prettier: true
+
+  deploy:
+    if: github.event.action == 'published'
+    name: Deploy Frontend to Heroku Production
+    runs-on: ubuntu-latest
+    timeout-minutes: 30
+    environment:
+      name: production
+      url: ${{ secrets.HEROKU_PROD_URL }}
```

actionlint (1.7.9)

context "secrets" is not allowed here. available contexts are "env", "github", "inputs", "job", "matrix", "needs", "runner", "steps", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)